### PR TITLE
[CDAP-7783] [CDAP-7859] Bugfix release/ui sort pagination filter search

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Fixes the exclusivity of sort and search query in search API

### CODE REVIEW

File: cdap-ui/app/cdap/components/EntityListView/index.js [hunk 1]
```diff
@@ -124,8 +131,13 @@ class EntityListView extends Component {
     }
 
     let query = nextProps.location.query || {};
-    let {filter = defaultFilter, q = '', page = 1, sort } = query;
-    let sortObj = !sort ? this.sortOptions[0] : this.sortOptions.find(sortOption => sortOption.sort === sort);
+    let {filter = defaultFilter, q = '', page = 1, sort, order } = query;
+    let sortObj;
+    if (typeof q === 'undefined'){
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/EntityListView/index.js:136]: can it be undefined? line 134 initialized `q` right?

File: cdap-ui/app/cdap/components/EntityListView/index.js [hunk 2]
```diff
@@ -124,8 +131,13 @@ class EntityListView extends Component {
...
+      sortObj = !sort ? this.sortOptions[0] : this.sortOptions.find(sortOption => sortOption.sort === sort && sortOption.order === order);
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/EntityListView/index.js]: also will this find the sort `none`?

File: cdap-ui/app/cdap/components/EntityListView/index.js (hunk 1 above)
Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/EntityListView/index.js:136]: I see good catch. Will fix that.

**Comment by elfenheart** [comment]
👍  LGTM

**Pull request merged**